docs/releases.rst

```diff
@@ -84,6 +84,9 @@ the ``clean`` subcommand. Users can clean resources from the last
 run or from a run with a given id. Users can also choose to skip
 cleaning ``guests``, ``runs`` or ``images``.
 
+``tmt try`` now supports :ref:`/stories/cli/try/option/install` option
```

Comment:
Needs to be moved to the right release.

Reply:
Done

Reply:
Is it? I see it in 1.35.0 release section, shouldn't it be in the upcoming one, 1.37.0?

Reply:
Hm, I believe that I changed that .. maybe due some rebase from other branch or change in format of release number it went back to 1.35 .. for now its fixed

Reply:
Seems to be covered.